csp/utils.py

```diff
@@ -114,6 +116,40 @@ def build_policy(config=None, update=None, replace=None, nonce=None):
                       for k, val in policy_parts.items()])
 
 
+def build_report_policy(config=None, update=None, replace=None):
```

Comment:
This could also go in a separate library.

Reply:
I've pulled the function.